### PR TITLE
Vmal

### PR DESCRIPTION
**Pull request by Malisevych**

### CODE REVIEW

File: RESTserver/src/main/java/com/softserve/if072/restservice/repository/ImageRepository.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+/**
+ * Created by Vitaliy Malisevych
+ */
+public interface ImageRepository extends JpaRepository<Image, Integer> {
+
+    List<Image> findAll();
```
Comment (Dyndyn) [inline RESTserver/src/main/java/com/softserve/if072/restservice/repository/ImageRepository.java:13]: MAJOR Add the "@Override" annotation above this method signature [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)

**Comment by Dyndyn** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/PdfCreatorService.java [hunk 2]
```diff
@@ -16,23 +11,13 @@
...
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
-import org.springframework.web.client.RestTemplate;
 
-import java.io.ByteArrayOutputStream;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```
Comment (kucherif) [inline MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/PdfCreatorService.java:16]: .

File: RESTserver/src/main/java/com/softserve/if072/restservice/repository/ImageRepository.java (hunk 1 above)
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/repository/ImageRepository.java:13]: this method is not needed. JPA provide it implicitly.

**Pull request merged**